server/services/team/service.ts

```diff
@@ -49,21 +50,31 @@ export class TeamService extends ServiceClass implements ITeamService {
 			_updatedAt: new Date(), // TODO how to avoid having to do this?
 		};
 
-		// create team
-		const result = await this.TeamModel.insertOne(teamData);
-		const teamId = result.insertedId;
+		let teamId: string;
+		try {
+			const result = await this.TeamModel.insertOne(teamData);
+			teamId = result.insertedId;
+		} catch (e) {
+			// TODO should we throw a generic error instead of saying that the team already exists?
+			if (e.code === 11000) {
```

Comment:
not sure what is the best, but we usually check the data before executing the `insert` instead of trying to guess the error.